### PR TITLE
Ray2 - Oid repair at ber

### PR DESCRIPTION
**Pull request by Martin22**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/discovery/ray.yaml [hunk 1]
```diff
@@ -37,7 +37,7 @@ modules:
             data:
                 -
                     oid: ber
-                    num_oid: '.1.3.6.1.4.1.33555.4.3.2.11.{{ $index }}'
+                    num_oid: '.1.3.6.1.4.1.33555.1.3.2.11..{{ $index }}'
```
Comment (PipoCanaja) [inline includes/definitions/discovery/ray.yaml:40]: You cannot have 2 dots in an OID. So this line must be wrong. Could you fix it  ?

**Review by PipoCanaja — CHANGES_REQUESTED** [review]

File: includes/definitions/discovery/ray.yaml [hunk 2]
```diff
@@ -37,7 +37,7 @@ modules:
             data:
                 -
                     oid: ber
-                    num_oid: '.1.3.6.1.4.1.33555.4.3.2.11.{{ $index }}'
+                    num_oid: '.1.3.6.1.4.1.33555.1.3.2.11.{{ $index }}'
```
Comment (PipoCanaja) [inline includes/definitions/discovery/ray.yaml:40]: Looks better for the OID. But looking at the MIB, it seems that a divisor is necessary here:
`1.3.6.1.4.1.33555.1.3.2.11: Bit Error Rate (BER) multiplied by 10^9. `

**Comment by PipoCanaja** [comment]
Whenever the divisor is corrected, you just need to run again the "save-test-data.php" script and you'll be good to go.

File: includes/definitions/discovery/ray.yaml (hunk 2 above)
Comment (Martin22) [reply includes/definitions/discovery/ray.yaml:40]: I don't think so. As far as I have seen graphs of other radio links, there was no divider used and the graphs showed the same values if there was a ber.

Comment (Martin22) [reply includes/definitions/discovery/ray.yaml:40]: The value is not divisible and is reported correctly.
https://www.techtarget.com/whatis/definition/bit-error-rate-BER

Comment (PipoCanaja) [reply includes/definitions/discovery/ray.yaml:40]: All right. Then you just need to generate again the JSON file and you should be good to go.

**Review by PipoCanaja — APPROVED** [review]
LGTM

**Pull request merged**